chainer_chemistry/datasets/citation_network/citeseer/.gitignore

```diff
@@ -0,0 +1,3 @@
+citeseer.cites
```

Comment:
remove this file. I think dataset should be inside `examples` directory, not inside library.

Reply:
fixed!